src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
Consider preparing the `usersFromServer` data in the `App` component and passing it down to child components instead of importing it directly. This will help maintain a clean data flow.